groups/ecs-service/main.tf

```diff
@@ -35,6 +35,9 @@ module "ecs-service" {
   healthcheck_path          = local.healthcheck_path
   healthcheck_matcher       = local.healthcheck_matcher
 
+  # ECS Task container health check
+  use_ecs_task_container_healthcheck = true
```

Comment:
Wasn't the name of the var changed to `use_task_container_healthcheck` (without ecs)?

Reply:
Yeah. Good point. Will push a new commit.